zipkin-storage/cassandra/src/main/resources/zipkin2-schema-indexes.cql

```diff
@@ -47,3 +47,17 @@ CREATE CUSTOM INDEX IF NOT EXISTS ON zipkin2.span (l_service) USING 'org.apache.
 
 CREATE CUSTOM INDEX IF NOT EXISTS ON zipkin2.trace_by_service_span (duration) USING 'org.apache.cassandra.index.sasi.SASIIndex'
    WITH OPTIONS = {'mode': 'PREFIX'};
+
+CREATE TABLE IF NOT EXISTS zipkin2.tags (
```

Comment:
I presume this is exact same as service-span

Reply:
Yep. Same as service-span